src/locales/ko/mystery-encounter-messages.json

```
@@ -1,6 +1,6 @@
 {
   "paid_money": "₽{{amount, number}}[[를]] 를 지불했다.",
   "receive_money": "₽{{amount, number}}[[를]] 받았다!",
-  "affects_pokedex": "Affects Pokédex Data",
-  "cancel_option": "Return to encounter option select."
+  "affects_pokedex": "Affects Pokédex Data--ME",
+  "cancel_option": "Return to encounter option select.ME"
```

```suggestion
  "affects_pokedex": "도감에 추가 가능",
  "cancel_option": "조우 선택지로 돌아가기"
```
affect pokedex 는 포켓몬 상인(충격) 에게서 포켓몬을 살때 해당 포켓몬이 도감에 새로 뭔가 추가됨( 숨특, 아마 이로치 )을 표현하는 아이콘에 커서를 올렸을때 나오는 overlay 메시지 입니다. https://github.com/AsdarDevelops/PokeRogue-Events/pull/112
cancel_option 은 Part-Timer and Dancing Lessons 등의 조우시 포켓몬 선택창에서 cancel 키에 hover 했을때 나오는 overlay 메시지 입니다. https://github.com/AsdarDevelops/PokeRogue-Events/pull/139